mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewclient/CreateNewClientFragment.java

```diff
@@ -252,7 +252,7 @@ public void onClickSubmitButton() {
             clientPayload.setExternalId(etClientExternalId.getEditableText().toString());
         }
 
-        if (!clientStaff.isEmpty()) {
+        if (!staffList.isEmpty()) {
```

Comment:
##Reason for app crash
**It is a java.lang.NullPointerException **   (as in offline mode clientStaff remains null)

As **clientStaff** represents List<Staff> and **staffList** represents List<String> (basically only names of staff for particular office)  
--In offline mode while creating a new client ,"showStaffInOffices(List<Staff> staffs)" method won't be called resulting in clientStaff to be null (as it was not initialized).

Thus, I checked for staffList (empty or not) within " onClickSubmitButton()" method ,because it is initialized when "oncreate()" method  called